README.Rmd

```diff
@@ -16,104 +14,91 @@ knitr::opts_chunk$set(
--   [Creating
-    individuals](https://www.open-systems-pharmacology.org/OSPSuite-R/articles/create-individual.html)
--   [Population
-    simulations](https://www.open-systems-pharmacology.org/OSPSuite-R/articles/create-run-population.html)
--   [PK
-    Analysis](https://www.open-systems-pharmacology.org/OSPSuite-R/articles/pk-analysis.html)
--   [Sensitivity
-    analysis](https://www.open-systems-pharmacology.org/OSPSuite-R/articles/sensitivity-analysis.html)
--   [Table
-    parameters](https://www.open-systems-pharmacology.org/OSPSuite-R/articles/table-parameters.html)
--   [Dimensions and
-    Units](https://www.open-systems-pharmacology.org/OSPSuite-R/articles/unit-conversion.html)
--   [Working with data sets and import from
-    excel](https://www.open-systems-pharmacology.org/OSPSuite-R/articles/observed-data.html)
--   [Working with `DataCombined`
-    class](https://www.open-systems-pharmacology.org/OSPSuite-R/articles/data-combined.html)
--   [Visualizations with
-    `DataCombined`](https://www.open-systems-pharmacology.org/OSPSuite-R/articles/data-combined-plotting.html)
+documentation](https://www.open-systems-pharmacology.org/OSPSuite-R/) for more
+details on the package.
+
+In particular, we would recommend that you read the articles in the following
+order:
+
+  - [Get Started](https://www.open-systems-pharmacology.org/OSPSuite-R/articles/ospsuite.html)
+  - [Loading a simulation and accessing entities](https://www.open-systems-pharmacology.org/OSPSuite-R/articles/load-get.html)
+  - [Changing parameter and molecule start values](https://www.open-systems-pharmacology.org/OSPSuite-R/articles/set-values.html)
+  - [Running a simulation](https://www.open-systems-pharmacology.org/OSPSuite-R/articles/run-simulation.html)
+  - [Efficient calculations](https://www.open-systems-pharmacology.org/OSPSuite-R/articles/efficient-calculations.html)
+  - [Creating individuals](https://www.open-systems-pharmacology.org/OSPSuite-R/articles/create-individual.html)
+  - [Population simulations](https://www.open-systems-pharmacology.org/OSPSuite-R/articles/create-run-population.html)
+  - [PK Analysis](https://www.open-systems-pharmacology.org/OSPSuite-R/articles/pk-analysis.html)
+  - [Sensitivity analysis](https://www.open-systems-pharmacology.org/OSPSuite-R/articles/sensitivity-analysis.html)
+  - [Table parameters](https://www.open-systems-pharmacology.org/OSPSuite-R/articles/table-parameters.html)
+  - [Dimensions and Units](https://www.open-systems-pharmacology.org/OSPSuite-R/articles/unit-conversion.html)
+  - [Working with data sets and import from excel](https://www.open-systems-pharmacology.org/OSPSuite-R/articles/observed-data.html)
+  - [Working with `DataCombined` class](https://www.open-systems-pharmacology.org/OSPSuite-R/articles/data-combined.html)
+  - [Visualizations with `DataCombined`](https://www.open-systems-pharmacology.org/OSPSuite-R/articles/data-combined-plotting.html)
```

Comment:
Add missing reference to the article https://www.open-systems-pharmacology.org/OSPSuite-R/dev/articles/PKSim-installation.html ?

Reply:
As PK-Sim installation is not required anymore, maybe we can remove the vignette ?

Reply:
At the moment, I wouldn't. It explains e.g. the difference between V11 and V12 (e.g. that you don't need a separate installation in V12 anymore).
I would keep the vignette for V12 - and remove it later for V13.